### DISCUSSION TITLE
Somehow the admin dashboard stopped working

### DISCUSSION THREAD

**Discussion by swijckmans** [avitorio/outstatic#110] [General]
Hey folks,

I don't want to log this as a bug yet but out of nowhere I noticed my outstatic project doesn't work correctly anymore. The admin dashboard '/outstatic' allows me to authenticate and after goes to a blank screen.

These are the errors I get (it doesn't show much). I have update my node version which may have caused something to break but not sure what.

<img width="986" alt="Screenshot 2023-06-18 at 22 23 58" src="https://github.com/avitorio/outstatic/assets/61318055/ad6b5224-f1f5-4fa1-a5f0-a9a97fe89c56">

**Comment by avitorio** [comment]
Hey,
I'm sorry about this. Please try setting `swcMinify: false` in your next.config.js

Let me know if it helps.

**Reply by avitorio** [reply]
https://github.com/avitorio/outstatic/issues/109

I've related this here 👆🏼


Have been struggling with it for the last few days 😢

**Reply by swijckmans** [reply]
I disabled SWC and called it a day for now. Excited to enable it again in the future when you find a solution.

Also greatly love the ability to add fields. I was missing that immensely.

**Reply by avitorio** [reply]
Hopefully we'll be able to re-enable swcMinify in the next version. 
Glad you like the custom fields feature!
